topics/building.adoc

```diff
@@ -1,7 +1,9 @@
 [[ccg-building-content]]
 == Building Content
 
-Placeholder text.
+Writing your documentation using AsciiDoc or other formats means that you can build HTML, PDF, EPUB, and man pages. During the writing process, you build your documentation to test and review the content. Depending on the complexity of the documentation, you might decide to adopt a tool that helps you organize the building of documentation sets, for example, AsciiBinder.
```

Comment:
We might want to clarify 'other formats'  - Should it be 'other documentation markup languages' instead? Or we could remove it altogether.

Reply:
Maybe instead of:

"Depending on the complexity of the documentation, you might decide to adopt a tool that helps you organize the building of documentation sets, for example, AsciiBinder."

We go with : 

"Depending on the complexity of the documentation, you might decide to adopt a tool that helps you organize the building of documentation sets. This section will introduce you to some of those tools."

Reply:
Note that we have `Rendering Content Locally` included in this chapter, and we can assume people always want to build their docs locally, regardless of whether they also adopt a tool to organize content, etc.

Reply:
@pmkovar @nengard  updated based on comments